### DISCUSSION TITLE
QST: Street centrality weighted by floor area within specific radius (meter)?

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#338] [Q&A]
I am a beginner using momepy that is an invaluable tool. 

Now, I am wondering if momepy can measure street centrality such as betweenness, closeness, reach, and straightness weighted by the floor area of buildings within a specific radius (meter) like Urban Network Analysis (UNA) created by MIT City Form Lab.

**Comment by martinfleis** [comment]
Hi @cdkang,

I am not sure. Do you know how does that weighting work? We may find a way of doing it as you can specify custom weights using the `weight` keyword. If we can populate it with right values before the computation, it should be possible.

**Comment by cdkang** [comment]
Thank you for your advice. Let me try.
